config.json

```diff
@@ -1 +0,0 @@
-{"common":{"log_level":"info","log_path":"/jdbc.log"}}
```

Comment:
This file was accidentally added in this PR: https://github.com/snowflakedb/snowflake-jdbc/commit/76e4efcdd3bb15ba7a7d83461c432f8a29816bf3